index.js

```diff
@@ -0,0 +1,52 @@
+/*eslint-disable */
+import { ui } from './modules/ui.js';
+import { store } from './modules/store.js';
+
+class Books {
+  constructor(Title, Author) {
+    this.title = Title;
+    this.author = Author;
+  }
+}
+
+// add book//
+document.addEventListener('DOMContentLoaded', ui.display());
+document.querySelector('#form').addEventListener('submit', (e) => {
+  const Title = document.querySelector('#title').value;
+  const Author = document.querySelector('#author').value;
+  const booksCollection = store.getBooksCollection();
+  const book = new Books(Title, Author);
+  store.setBook(book);
+});
+// delete Book//
+const remove = document.querySelector('.bookList');
+remove.addEventListener('click', (e) => {
+  store.deleteBooks(e.target.previousElementSibling.firstElementChild);
+  location.reload();
+});
+
+const List = document.querySelector('.List');
+const Add = document.querySelector('.Add');
+const Contact = document.querySelector('.Contact');
+
+Add.addEventListener('click', () => {
+  document.querySelector('.book-list-container').style.display = 'none';
+  document.querySelector('.input').style.display = 'block';
+  document.querySelector('.contact').style.display = 'none';
+});
+
+List.addEventListener('click', () => {
+  document.querySelector('.book-list-container').style.display = 'block';
+  document.querySelector('.input').style.display = 'none';
+  document.querySelector('.contact').style.display = 'none';
+  location.reload();
+});
+
+Contact.addEventListener('click', () => {
+  document.querySelector('.book-list-container').style.display = 'none';
+  document.querySelector('.input').style.display = 'none';
+  document.querySelector('.contact').style.display = 'block';
+});
+const displaytime = document.querySelector('.time');
+const shownDate = [Date().split(' ').splice(1, 4).join(' ')];
+displaytime.append(shownDate);
```

Comment:
- [ ] As pointed out by the previous reviewer and based on the project requirements, we are supposed to use the Luxon library to display the current date. You will be required to add a full or minified package of the Luxon library and then import the module `import { DateTime } from "luxon";`
- use [this Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) documentation on how to set it up on your project.